src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -421,6 +503,14 @@ private sun.security.provider.SecureRandom getMixRandom() {
         private static void readFully(InputStream in, byte[] data)
                 throws IOException {
             int len = data.length;
+            /*[IF CRIU_SUPPORT]*/
+            if (flag == Variant.CRIU) {
+                // read bytes from non blocking source
+                if (in.readNBytes(data, 0, len) < len) {
+                    throw new IOException("Could not read from file(s)");
+                }
+            }
+            /*[ELSE] CRIU_SUPPORT*/
```

Comment:
Please ensure there's a space before `*/` on all `[ELSE]` and `[ENDIF]` lines.

Reply:
updated